### PR TITLE
chore: add dataconsumerTwo, pgadmin4 and improve portal

### PR DESCRIPTION
**Pull request by evegufy**
## Description

- add dataconsumerTwo: disabled by default, to be installed at upgrade of data exchange subset
required https://github.com/eclipse-tractusx/tractus-x-umbrella/pull/81
- add pgadmin4 instance for database access
- change to portal 1.8.1 and complete config

https://github.com/eclipse-tractusx/sig-release/issues/418

## Pre-review checks

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: charts/umbrella/README.md [hunk 1]
```diff
@@ -344,6 +368,24 @@ helm delete umbrella --namespace umbrella
 > If persistance for one or more components is enabled, the persistent volume claims (PVCs) and connected persistent volumes (PVs) need to be removed manually even if you deleted the release from the cluster.
 >
 
+### Database Access
+
+This chart also contains a pgadmin4 instance for easy access to the deployed Postgres databases which are only available from within the Kubernetes cluster.
+
+pgadmin4 is by default enabled with in the predefined subsets for data exchange and portal.
+
+Address: [pgadmin4.tx.test](http://pgadmin.tx.test)
```
Comment (almadigabor) [inline charts/umbrella/README.md:377]: I think it should be either `pgadmin4.tx.test` or `pgadmin.tx.test`. Not the text and link are different.

Comment (evegufy) [reply charts/umbrella/README.md]: thanks, I corrected it

**Review by almadigabor — APPROVED** [review]
LGTM

**Review by ds-jhartmann — APPROVED** [review]
LGTM

**Pull request merged**